framework/console/controllers/ServeController.php

```diff
@@ -52,6 +52,7 @@ class ServeController extends Controller
     public function actionIndex($address = 'localhost')
     {
         $documentRoot = Yii::getAlias($this->docroot);
+        $router = Yii::getAlias($this->router);
```

Comment:
Shouldn't we check if `$this->router !== null` before using `Yii::getAlias()`?

Reply:
Wouldn't the same logic apply for `$this->docroot`? I went for consistency. Should we check for both or just the router?

Reply:
`docroot` has default value, which is already a string, and from console you can overwrite it only by another string. `router` is `null` by default, so we should expect that it will stay as `null`. So IMO this check is needed only for `router`.